src/internal/Popover.js

```diff
@@ -449,24 +381,24 @@ class Popover extends Component<DefaultProps, AllProps, void> {
       ...other
     } = this.props;
 
+    // FIXME: props API consistency problem? - `...other` not spread over the root
```

Comment:
I agree, let's move the ...other to the root. What's preventing us from doing it?